src/server/database/documents/Life.ts

```diff
@@ -0,0 +1,11 @@
+import { ObjectId } from 'mongodb';
+
+export type Life = {
+    _id: ObjectId;
+    firstName: string;
+    lastName: string;
+    birthday: Date;
+    title: string;
+    description: string;
+    hobbies: Array<string>;
```

Comment:
for `Array`s, usually we use `[]`, so `string[]`;

one exception is to nest object inside, say `Array<{ hello: string ; world: number }>`, for this we tend to use `Array`